src/Compilers/VisualBasic/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.vb

```diff
@@ -556,6 +556,163 @@ True
 ]]>
 
             Dim verifier As CompilationVerifier = CompileAndVerify(source, expectedOutput)
```

Comment:
That would only throw when the said code get executed more than once, with actual runtime. The test code is compiled against mock runtime and never executed.